packages/stylelint-plugin-mozaic/lib/checkCase.js

```diff
@@ -0,0 +1,45 @@
+var stylelint = require('stylelint')
+
+const cases = {
+  'kebab-case': /^([a-z][a-z0-9]*)(-[a-z0-9]+)*$/,
+  lowerCamelCase: /^[a-z][a-zA-Z0-9]+$/,
+  snake_case: /^[a-z][a-zA-Z0-9]+$/,
+  PascalCase: /^[A-Z][a-zA-Z0-9]+$/,
+  UpperCamelCase: /^[A-Z][a-zA-Z0-9]+$/,
```

Comment:
no differences between pascal case and UpperCamelCase, the real name is pascal case.